lucene/test-framework/src/java/org/apache/lucene/index/BaseDocValuesFormatTestCase.java

```diff
@@ -1204,6 +1204,12 @@ public void testRandomSortedBytes() throws IOException {
   }
 
   private void doTestNumericsVsStoredFields(double density, LongSupplier longs) throws Exception {
+    doTestNumericsVsStoredFields(density, longs, 256);
+    // TODO: 200000 docs are needed to test jumps properly (see LUCENE-8585), but that is quite slow (few minutes).
+    // Maybe it can be nightly?
+    //doTestNumericsVsStoredFields(density, longs, 200000);
```

Comment:
Should we add a `if (TEST_NIGHTLY) ...`-switch here to to enable the 200K test case? It would increase coverage for jumps a great deal.

Reply:
Let's not do that in the base test case as this will be too slow with some codecs such as the SimpleTextCodec. What we have done before for formats that have branches that only get triggered with many documens is that we added tests to their own test file, ie. TestLucene80DocValuesFormat.

Reply:
By the way if we want to test such large numbers of documents, we should avoid RandomIndexWriter (use IndexWriter instead) and set reasonable values of maxBufferedDocs+RAMSizeBufferMB+mergePolicy instead of relying on random values that are set by LuceneTestCase#newIndexWriterConfig and make indexing slow (but are useful to increase coverage of flushing+merging)